### ISSUE TITLE
PrintClient with feature.data - print runs on an error if the feature.data column does not exist

### ISSUE THREAD

**Issue by astroidex** [mapbender/mapbender#692]
* if you define a feature.data definition in your print template f.e. feature.hauptcode that does not exist in the feature type, the print exits the pdf generation with an error
* in this case feature.hauptcode is written with a k feature.hauptkode

`CRITICAL - Uncaught PHP Exception Symfony\Component\Debug\Exception\ContextErrorException: "Notice: Undefined index: hauptcode" at /data/git/mainz/application/vendor/mapbender/data-source/Entity/DataItem.php line 158 `

ToDo: Ignore is feature.column is not present in the feature type

**Comment by astroidex** [comment]
The errors in vendor/mapbender/data-source/Entity/DataItem.php at line 159  -

```
Notice: Undefined index: abstractNIX
500 Internal Server Error - ContextErrorException 
```

FIX could be error_reporting(0);

```
    public function getAttribute($name){
        $attributes = $this->getAttributes();
        error_reporting(0);
        return $attributes[$name];
    }
```

**Comment by astroidex** [comment]
* fixed with pull reuqest #701
